### DISCUSSION TITLE
Is jelly vue the future?

### DISCUSSION THREAD

**Discussion by Sparklingx** [jellyfin/jellyfin-vue#2039] [Questions] [answered]
Is this the future? so far loving it.  right now the default jellyfin webui player is laggy when seeking using key, when I press forward or backward key its laggy or stack for a moment. jelly vue works flawlessly ....  btw what is the  key binding for volume up or down? up or down arrow key does not work.

**Comment by ferferga** [comment, marked as answer]
@Sparklingx No, it's an alternative client to Jellyfin Web. The only plan we have related to "replacing" Jellyfin Web is to, in the near future, provide it as a separate package, so users can choose whether they want Vue or current web as their frontend.

This client doesn't focus at all in backwards compatibility for instance, so Jellyfin Web will still be rocking for those devices.

About the volume keys, they're simply not implemented yet, but will do once my exams are over 👍🏻.

**Reply by Sparklingx** [reply]
im guessing  changing quality option is not implemented yet too? 
![image](https://github.com/jellyfin/jellyfin-vue/assets/85538352/62617420-47d7-4fb4-bd5c-35721cc65d71)

**Comment by HammyHavoc** [comment]
> @Sparklingx No, it's an alternative client to Jellyfin Web. The only plan we have related to "replacing" Jellyfin Web is to, in the near future, provide it as a separate package, so users can choose whether they want Vue or current web as their frontend.
> 
> This client doesn't focus at all in backwards compatibility for instance, so Jellyfin Web will still be rocking for those devices.
> 
> About the volume keys, they're simply not implemented yet, but will do once my exams are over 👍🏻. 

Would be cool if the Jellyfin server could load it as an option for a specific client, but not all of them, i.e., if it isn't compatible.

**Reply by ferferga** [reply]
This is the approach I'd like too!
